src/main/java/io/github/intoto/slsa/models/v02/Metadata.java

```diff
@@ -0,0 +1,101 @@
+package io.github.intoto.slsa.models.v02;
+
+import com.fasterxml.jackson.annotation.JsonFormat;
+import java.time.OffsetDateTime;
+import java.util.Objects;
+
+/** Other properties of the build. */
+public class Metadata {
+
+  /**
+   * Identifies this particular build invocation, which can be useful for finding associated logs or
+   * other ad-hoc analysis. The exact meaning and format is defined by builder.id; by default it is
+   * treated as opaque and case-sensitive. The value SHOULD be globally unique.
+   */
+  private String buildInvocationId;
+
+  /**
+   * The timestamp of when the build started. A point in time, represented as a string in RFC 3339
+   * format in the UTC time zone ("Z").
+   */
+  @JsonFormat(shape = JsonFormat.Shape.STRING, pattern = "yyyy-MM-dd'T'HH:mm:ssXXX")
```

Comment:
I wonder if you want to promote this to a const at the top of the module so you can use it all throughout

Reply:
bump, @lakshya8066